src/_guides/openapi/code-first.md

```diff
@@ -0,0 +1,177 @@
+---
+title: "Code-first: How to Generate OpenAPI Descriptions"
```

Comment:
We have chosen to use `document` when talking about the file, `definition` when talking about the API structure. I'm not against using `description` here, but as it's part of our guides, I think it would be great to be consistent. WDYT?

Reply:
API Description = the contract that it lives in in whatever theoretical form.
API Description Document = the actual file it lives in, if it even lives in a file.
API Description Format = the format the file is in (OpenAPI, JSON Schema, AsyncAPI)

So normally when I'm saying OpenAPI Description I mean the whole concept of trying to make a description for the API, and I only mention the document when the fact that its a file seems important.

Reply:
Could just do `Code-first: How to Generate OpenAPI from Code`